### ISSUE TITLE
No count on region facet

### ISSUE THREAD

**Issue by popzip** [marklogic/eauser-geomapper#222]
All other facets (EA Version, Companies, Features) display the count next to each facet option.
Would be nice to have this for region as well. Not sure if bug or enhancement.

**Comment by kmcostel** [comment]
The code may have been changed, but I believe this would be an enhancement. It was never intended to be a facet but rather a simple geoquery on MarkLogic. I would run each query on page load and use the estimate as a count?

**Comment by dmcassel** [comment]
Yup, that's an enhancement. I agree it would be useful.

**Comment by dmcassel** [comment]
Generating facets on geospatial regions isn't as simple as for other values. The standard approach is to use cts:geospatial-boxes(), but that won't match up well with the regions we have defined. 

Options:

- there are only six regions and a fairly small number of documents, so go ahead and do the searches to get the counts
- get the counts as above, but use bounding boxes to cut down the candidates for each region
- on update, set a JSON property that identifies the region statically. This would perform well, but risks the static values and regions being out of sync if the regions get changed

**Comment by popzip** [comment]
It's not terribly important - I can see the counts for the regions if I click on the checkbox. Would just be nice to know before clicking. I guess the reason you can't easily display although you can easily filter is due to the background query needed to get the count without doing the search?

Option 3 would work but probably a worse design in terms of being inflexible.

**Comment by dmcassel** [comment]
Correct. Given that the data set is so small, we can probably get away with doing the searches, though.
